frontend/src/app/(routes)/governance/AllProposals.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+import React from 'react';
+import Image from 'next/image';
+import './style.css';
+
+const AllProposals = () => {
+  const proposalData = [
```

Comment:
When working with dummy data, it is recommended to create a JSON file and use its URL instead of the actual API URL. By doing this, you can easily integrate the data with your application without making any extra changes, apart from the API URL. Simply get the data to the state and use it as necessary. This approach will make the integration process smoother and easier.